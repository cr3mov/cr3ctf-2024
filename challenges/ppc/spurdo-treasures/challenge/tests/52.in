999999999000000000 1000000000000000000
