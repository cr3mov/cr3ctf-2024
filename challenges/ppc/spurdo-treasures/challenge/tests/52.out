999999999000000000 999999999944727844
